Reqnroll/Formatters/FormatterPluginBase.cs

```diff
@@ -0,0 +1,196 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using Io.Cucumber.Messages.Types;
+using Reqnroll.Formatters.Configuration;
+using Reqnroll.Formatters.PayloadProcessing.Cucumber;
+using Reqnroll.Formatters.PubSub;
+using Reqnroll.Formatters.RuntimeSupport;
+using Reqnroll.Plugins;
+using Reqnroll.UnitTestProvider;
+
+namespace Reqnroll.Formatters;
+
+public abstract class FormatterPluginBase : ICucumberMessageSink, IDisposable
```

Comment:
As these are not plugins anymore (which is good), I would rename the `XXXFormatterPluginYYY` classes to `XXXFormatterYYY`. E.g. `FormatterBase`, `HtmlFormatter`, etc.

Reply:
Completed.